.github/workflows/build-unix.yml

```diff
@@ -43,6 +43,13 @@ jobs:
             configure_opts: "--with-libewf --with-libqcow --with-libvhdi --with-libvmdk --disable-java CC=clang CXX=clang++"
             codecov: "no"
 
+          - os: "linux"
+            arch: "x86_64"
+            linkage: "static"
+            runner: "ubuntu-22.04"
+            configure_opts: "--enable-static --disable-shared --disable-java --without-afflib --without-libcrypto --without-libewf --without-libvhdi --without-libvmdk"
```

Comment:
Why are you disabling these libraries? Don't we want them for unit testing?

Reply:
this is the build I use of OssFuzz, if we want them we we'll need to build them statically